content/about/governance.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Governance"
+date: 2024-08-27T10:12:05+02:00
+layout: "general"
+---
+
+# Governance
+The Project Steering Committee guides the GRASS project’s direction, budget, and
+governance, making decisions through voting and overseeing the Developer Team
```

Comment:
```suggestion
governance, making decisions through voting and overseeing the Development Team
```

I think we commonly call it "GRASS Development Team".

Reply:
Good catch. By the way, how would you define it?

Reply:
Suggestion:

The GRASS Development Team is an international group of developers and scientists from various fields who work together to continuously improve the capabilities of GRASS GIS. Its diverse composition allows a wide range of expertise to be applied to the development of the software.